scripts/collect.sh

```diff
@@ -86,15 +91,19 @@ done
 # remove the initial connector
 install="${install# & }"
 configure="${configure# & }"
+known="${known# | }"
 
 echo >&2 "install:   $install"
 echo >&2 "configure: $configure"
+echo >&2 "known:     $known"
 
 # escape the connector for sed: & -> \&
 install="${install//&/\\&}"
 configure="${configure//&/\\&}"
+known="${known//|/\\|}"
```

Comment:
This may work but it is wrong: the `&` character is special in the *replacement* part of a substitution (meaning "the whole matched string") which is why we escape it. OTOH `|` is special only in the *match* part of `sed` where `known` is not used.

Reply:
Yes, it worked but you are right. There is not need for escaping it.